webpack.prod.js

```diff
@@ -3,14 +3,14 @@ const common = require('./webpack.common.js')
 const TerserPlugin = require('terser-webpack-plugin')
 
 module.exports = merge(common, {
+  devtool: 'eval',
```

Comment:
terser update: removed the sourceMap option (respect the devtool option by default)
So I just added the config from dev, not sure if that's right?